src/components/NotificationModal/index.tsx

```diff
@@ -7,13 +7,15 @@ import styled from "@emotion/styled";
 import { Body } from "@leafygreen-ui/typography";
 import get from "lodash/get";
 import set from "lodash/set";
-import { SubscriptionMethod } from "types/subscription";
+import { SubscriptionMethodDropdownOption } from "types/subscription";
 import { v4 as uuid } from "uuid";
 import { useBannerDispatchContext } from "context/banners";
+import Icon from "@leafygreen-ui/icon";
 import {
   useNotificationModal,
   UseNotificationModalProps,
 } from "hooks/useNotificationModal";
+import { RegexSelectorInput } from "components/NotificationModal/RegexSelectorInput";
```

Comment:
can this just be a relative import since they are in the same directory?

Reply:
i don't think imports should ever be relative

Reply:
i also prefer to use absolute imports because the import string is valid from any file in our src folder

Reply:
makes sense!